focus-ios/Blockzilla.xcodeproj/project.pbxproj

```diff
@@ -4169,6 +4254,529 @@
 			};
 			name = KlarEnterprise;
 		};
+		451B81C22BA4DB9F00CF3C50 /* KlarDebug */ = {
```

Comment:
@isabelrios Do we use xcconfig file? (cc @nbhasin2 )

Reply:
As far as I know, we don't use it...